src/Juvix/Compiler/Core/Extra/Base.hs

```diff
@@ -192,6 +192,12 @@ mkDynamic i = NDyn (Dynamic i)
 mkDynamic' :: Type
 mkDynamic' = mkDynamic Info.empty
 
+mkBottom :: Info -> Type
+mkBottom i = NBot (Bottom i)
+
+mkBottom' :: Type
+mkBottom' = NBot (Bottom mempty)
```

Comment:
```suggestion
mkBottom' :: Type
mkBottom' = mkBottom mempty
```
However, I think you should remove this.

Reply:
I've applied your suggestion.

why do you think I should remove it?
we are following the same pattern for every other `Node`.

Reply:
Reason: the last time I searched for references to `mkBottom` there were none.

Reply:
It might make sense to keep it though, given that it's the "universal" interface for nodes. One would naturally expect the primed version to be available, which provides the default empty info.

I'm not sure if all other primed versions are used, but they're there in case someone needs them.